keepassxc-browser/content/username-field.js

```diff
@@ -123,19 +123,8 @@ const iconClicked = async function(field, icon) {
         return;
     }
 
-    let reconnected = false;
-
     // Try to reconnect if KeePassXC is not currently connected
-    const connected = await sendMessage('is_connected');
-    if (!connected) {
-        const reconnectResponse = await sendMessage('reconnect');
-        if (!reconnectResponse.keePassXCAvailable) {
-            kpxcUI.createNotification('error', tr('errorNotConnected'));
-            return;
-        }
-
-        reconnected = true;
-    }
+    let reconnected = await reconnect();
```

Comment:
This is not working as intended. Please see the original function.

Reply:
oof, I'll look into it again 🙃